File: Hilbert - Unity/Assets/Main/Scripts/LoadScreen.py
import UnityEngine as ue
import time

class LoadScreen():
    def __init__(xWaitTime, xSceneName):
        self.waitTime = xWaitTime
        self.waitTime = xSceneName
        #default is 3 seconds

    def LoadScene(sceneName):
        #sceneName is a string, the scene you want to load.
        time.sleep(xWaitTime)
        SceneManager.LoadScene(sceneName);

File: Hilbert - Unity/Assets/Scripts/LoadScreen.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.SceneManagement;

public class LoadScreen : MonoBehaviour
{
    public int waitTime; //Delay in changing Scenes
    public void LoadScene(string sceneName){
        //Add Animation here if needed
        StartCoroutine(WaitForSceneLoad(sceneName));
    }
    private IEnumerator WaitForSceneLoad(string sceneName){
        yield return new WaitForSeconds(waitTime);
        SceneManager.LoadScene(sceneName);
    }
    public void QuitGame()
    {
        Application.Quit();
    }
}
